libnetwork/cni/config.go

```diff
@@ -17,7 +17,7 @@ import (
 
 // NetworkCreate will take a partial filled Network and fill the
 // missing fields. It creates the Network and returns the full Network.
-func (n *cniNetwork) NetworkCreate(net types.Network) (types.Network, error) {
+func (n *cniNetwork) NetworkCreate(net types.Network, options types.NetworkCreateOptions) (types.Network, error) {
```

Comment:
I think this should be a pointer.  That's what we (more or less) consistently do in Podman-land.

```suggestion
func (n *cniNetwork) NetworkCreate(net types.Network, options *types.NetworkCreateOptions) (types.Network, error) {
```

Reply:
Done